wallet/wallet.go

```diff
@@ -2781,6 +2781,15 @@ func (w *Wallet) resendUnminedTxs() {
 	for _, tx := range txs {
 		resp, err := chainClient.SendRawTransaction(tx, false)
 		if err != nil {
+			// If the transaction has already been accepted into the
+			// mempool, we can continue without logging the error.
+			switch {
```

Comment:
This doesn't seem to address the actual issue: mistaking a double spend, or an error that the transaction has already been included in the chain.

Reply:
Ah, I see the TODO below